### DISCUSSION TITLE
2.0.22版本出现一个BUG，无法使用Zotero笔记自带的引注功能

### DISCUSSION THREAD

**Discussion by zhu-zhiyun** [windingwind/zotero-better-notes#1190] [Q&A]
更新新版本BN后，Zotero自带的引注功能不能用了。
![条目显示](https://github.com/user-attachments/assets/b044ddbe-e3e2-4752-bf7a-6368badd7337)
新版本之前，插入引注，单击引注信息，会出现编辑引注等选项。
新版本更新后，单击没作用了。

**Comment by windingwind** [comment]
Update plugin

**Reply by zhu-zhiyun** [reply]
更新后，问题解决。👍
